### DISCUSSION TITLE
Anyone else been contacted by tkm.gmbh or Bela Wiertz

### DISCUSSION THREAD

**Discussion by gene1wood** [netbirdio/netbird#586] [Issue Triage]
I got an email today about Netbird. Not sure if this is a scam or some kind of investment entity looking at investing in Netbird. Has anyone else gotten an email like this after starring the Netbird repo?

> ---------- Forwarded message ---------
> From: Bela Wiertz <>
> Date: Fri, Nov 25, 2022 at 4:02 AM
> Subject: Question about Netbird's Repository/Technology
> To:
> 
> Hey,
> 
> Maybe a strange request, but I have seen, that you starred Netbirds main Repo on GitHub.
> Have you tried it? If yes, Why? and how did you liked their technology?
> Would be great If you could give me some feedback.
> 
> If you have questions back to me, do not hesitate.
> 
> Thank you a lot,
> ______________________
> 
> Bela Wiertz
> Investment Team
> 
> TK Management GmbH
> M: +XX XX XXXXXXXXX
> E: xxx@xxxx.xxx
> A: XXXXXX | XXXXXX
> 
> Amtsgericht Köln HRB 91308
> Geschäftsführer: Thomas Kresing
> 
> Diese E-Mail und etwa anhängende Dateien enthalten vertrauliche Informationen und sind ausschließlich für den Adressaten bestimmt. Sollten Sie diese E-Mail irrtümlich erhalten haben, löschen Sie diese E-Mail einschließlich etwa angehängter Dateien aus Ihrem System. Bitte beachten Sie, dass jede Form der Kenntnisnahme, Veröffentlichung oder Weitergabe der E-Mail und etwa angehängter Dateien verboten ist.

Note the `To` field is blank and the email was BCCd to me. Searching GitHub, maybe this is @wiertzbl as the user has the same name and has starred the Netbird repo

**Comment by mlsmaycon** [comment]
Hello @gene1wood, thanks for letting us know about the email. 

Yes, we are in touch with them, but we didn't know they would be contacting users who have starred our project. 

If this was an issue, we apologize and will discuss this approach with them.

**Comment by gene1wood** [comment]
> Yes, we are in touch with them

Ok, great that confirms that the email is legit

> If this was an issue, we apologize and will discuss this approach with them.

No, not an issue, other than I couldn't tell if it was a legitimate email. 

It was vague enough that it read like the beginning of a scam. If TK Management is looking to reach out to users of Netbird, I'd recommend you have them provide more detail in their email, perhaps have a domain name for their email addresses with a website attached, maybe link to a page on the netbird.io site that confirms that they're contacting users. That sort of thing.

Additionally, just to follow good netiquette, they should provide a pathway for users to opt-out of being communicated with.

**Reply by mlsmaycon** [reply]
Thank you @gene1wood for your suggestions, they make a lot of sense and we will discuss them with team at TK management.
